app/src/main/java/org/greenstand/android/TreeTracker/database/entity/PlanterInfoEntity.kt

```diff
@@ -27,7 +27,9 @@ data class PlanterInfoEntity(
     @ColumnInfo(name = CREATED_AT)
     var createdAt: Long,
     @ColumnInfo(name = BUNDLE_ID)
-    var bundleId: String? = null
+    var bundleId: String? = null,
+    @ColumnInfo(name = RECORD_UUID)
+    var recordUuid: String? = null
```

Comment:
When would the `recordUuid` will be null?  Won't it be a non-null value always?

Reply:
Updated to non null